aten/src/ATen/native/native_functions.yaml

```diff
@@ -7768,28 +7768,28 @@
     Meta: normal_meta_
```

Comment:
Why is this one not ported to structured kernels as well? I reckon that we should have all the combinations of functions here (in-place / out-place / _out) for all the types of inputs (Tensor / float) for (mean / std), right?

Reply:
i'll look into it and follow up later. at first, it looked like it wasn't possible for some reason. maybe i just got confused